website/public/site.webmanifest

```diff
@@ -1,33 +1,19 @@
 {
   "name": "Marquee",
   "short_name": "Marquee",
-  "start_url": "/",
-  "display": "standalone",
-  "orientation": "portrait",
-  "theme_color": "#ffffff",
-  "background_color": "#ffffff",
   "icons": [
     {
-      "src": "/favicon/android-chrome-192x192.png",
+      "src": "favicon/android-chrome-192x192.png",
       "sizes": "192x192",
       "type": "image/png"
     },
     {
-      "src": "/favicon/android-chrome-512x512.png",
+      "src": "favicon/android-chrome-512x512.png",
       "sizes": "512x512",
       "type": "image/png"
-    },
-    {
-      "src": "/images/appicons/maskable_icon.png",
```

Comment:
Do we have no replacement for it? I've been using https://maskable.app/editor to generate these.

Reply:
The path was wrong so I deleted it altogether. Brought it back with a working version, though.

Reply:
👍